features/is-analytics-solution-features/org.wso2.carbon.sp.solutions.is.analytics.feature/pom.xml

```diff
@@ -0,0 +1,205 @@
+ ~
+ ~ http://www.apache.org/licenses/LICENSE-2.0
+ ~
+ ~ Unless required by applicable law or agreed to in writing, software
+ ~ distributed under the License is distributed on an "AS IS" BASIS,
+ ~ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ ~ See the License for the specific language governing permissions and
+ ~ limitations under the License.
+ -->
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <parent>
+        <artifactId>is-analytics-solution-features</artifactId>
+        <groupId>org.wso2.analytics.solutions</groupId>
+        <version>1.0.0-SNAPSHOT</version>
+    </parent>
+    <modelVersion>4.0.0</modelVersion>
+    <name>WSO2 Stream Processor - WSO2 Identity Server Analytics Solution Feature</name>
+    <artifactId>org.wso2.carbon.sp.solutions.is.analytics.feature</artifactId>
+
+    <dependencies>
+        <!-- Widgets-->
+        <dependency>
+            <groupId>org.wso2.analytics.solutions</groupId>
+            <artifactId>org.wso2.analytics.is.analytics.solutions.widgets.date.time.range.picker</artifactId>
+            <type>zip</type>
+        </dependency>
+        <dependency>
+            <groupId>org.wso2.analytics.solutions</groupId>
+            <artifactId>org.wso2.analytics.is.analytics.solutions.widgets.area.chart</artifactId>
+            <type>zip</type>
+        </dependency>
+        <dependency>
+            <groupId>org.wso2.analytics.solutions</groupId>
+            <artifactId>org.wso2.analytics.is.analytics.solutions.widgets.attempts.over.time</artifactId>
+            <type>zip</type>
+        </dependency>
+        <dependency>
+            <groupId>org.wso2.analytics.solutions</groupId>
+            <artifactId>org.wso2.analytics.is.analytics.solutions.widgets.compact.summary</artifactId>
+            <type>zip</type>
+        </dependency>
+        <dependency>
+            <groupId>org.wso2.analytics.solutions</groupId>
+            <artifactId>org.wso2.analytics.is.analytics.solutions.widgets.horizontal.bar.chart</artifactId>
+            <type>zip</type>
+        </dependency>
+        <dependency>
+            <groupId>org.wso2.analytics.solutions</groupId>
+            <artifactId>org.wso2.analytics.is.analytics.solutions.widgets.messages</artifactId>
+            <type>zip</type>
+        </dependency>
+        <dependency>
+            <groupId>org.wso2.analytics.solutions</groupId>
+            <artifactId>org.wso2.analytics.is.analytics.solutions.widgets.summary</artifactId>
+            <type>zip</type>
+        </dependency>
+        <dependency>
+            <groupId>org.wso2.analytics.solutions</groupId>
+            <artifactId>org.wso2.analytics.is.analytics.solutions.widgets.user.preferences</artifactId>
+            <type>zip</type>
+        </dependency>
+        <dependency>
```

Comment:
Do we need a testng dependency in the feature?

Reply:
No, will remove it